hq/app/aws/AWS.scala

```diff
@@ -20,4 +33,39 @@ object AWS {
       Failure.awsAccountNotFound(accountId).attempt
     )
   }
+
+  private def clients[T, S <: AwsClientBuilder[S, T]](
```

Comment:
Maybe be good to have a test for this? Make sure it actually creates clients for the provided regions.

Reply:
Done.